### ISSUE TITLE
💻remove cheatsheets

### ISSUE THREAD

**Issue by AnneliesVlaar** [hedyorg/hedy#6022]
As suggested in the [roadmap](https://github.com/hedyorg/hedy/wiki/Roadmap) it might be better to remove the cheatsheets. They are not user friendly see: ' Yellow paste-this-code button in Cheatsheet' #5652 and  'Inaccessible items (for keyboard users)' #2925

But maybe there is still some need for teachers to have some kind of cheatsheet see: 'Add printable cheatsheets to the teacher manual' #4135

**Comment by Felienne** [comment]
Yes the printable cheats would be lovely, maybe in a different form though (with small example programs so teachers can quickly look up how it works)

**Comment by AnneliesVlaar** [comment]
> Yes the printable cheats would be lovely, maybe in a different form though (with small example programs so teachers can quickly look up how it works)

Would something like this be helpful:

```
# level 1
print some text or number 1
ask a question?
echo the answer to the question

# make some music with
play C4
play 28
# turn the turle
turn left
turn right
# move the turtle
forward 100
```

```
# level 2
print some text or numbers 123
answer is ask a question?
print the answer to the question
variable is a text or a number 123

# make some music with
play C4
play 28
# turn the turle
turn 360
# move the turtle
forward 100
# wait for a second
sleep 1
```

```
# level 3
# print, ask, variables
print some text or numbers 123
answer is ask a question?
print the answer to the question
variable is a text or a number 123

# list
list is text, or, a, number, 123
print list at random
add something to list
remove something from list

# music, turtle, sleep
# make some music with
play C4
play 28
# turn the turle
turn 360
# move the turtle
forward 100
# wait for a second
sleep 1
```

```
# level 4
# print, ask, variables
print 'some text' 'or numbers 123'
answer is ask 'a question?'
print 'the' answer 'to the question'
variable is 'a text or a: '
number is 123

# list
list is text, or, a, number, 123
print list at random
add something to list
remove something from list

# music, turtle, sleep, clear
# make some music with
play C4
play 28
# turn the turle
turn 360
# move the turtle
forward 100
# give color to the turtle
color green # var is green; color var
# wait for a second
sleep 1
# clear the output screen
clear
```

**Comment by MarleenGilsing** [comment]
I have made these printable cheatsheets once upon a time. They are probably slightly outdated now though. 

Maybe we can integrate them into the workbooks?

**Comment by AnneliesVlaar** [comment]
> I have made these printable cheatsheets once upon a time. They are probably slightly outdated now though.
> 
> Maybe we can integrate them into the workbooks?

create an assignment to have the students make their own cheatsheets?

**Comment by MarleenGilsing** [comment]
I was thinking about making it for them, but sure, also a good idea!

**Comment by Felienne** [comment]
> > I have made these printable cheatsheets once upon a time. They are probably slightly outdated now though.
> > Maybe we can integrate them into the workbooks?
> 
> create an assignment to have the students make their own cheatsheets?

That is a lovely idea!! Something like: write down the codes you always forget.

But I don't this it is either or!

**Comment by MarleenGilsing** [comment]
**For the hackathon** 

<img width="821" height="773" alt="Image" src="https://github.com/user-attachments/assets/26d17ecd-26a6-4eb1-acc8-a43f485e70a1" />

We want an cheatsheet adventure that opens the cheatsheet of the corresponding level in a new tab. The cheatsheet is not a normal adventure, so it should have an icon to indicate this. Making it an adventure automatically makes it easy for a teacher to disable them.
